Makefile

```diff
@@ -252,6 +246,10 @@ test-unit: $(SETUP_ENVTEST) envtest-k8s-bins #HELP Run the unit tests
                 $(UNIT_TEST_DIRS) \
                 -test.gocoverdir=$(COVERAGE_UNIT_DIR)
 
+.PHONY: test-regression
+test-regression: #HELP Run regression test
+	go test -count=1 -v ./test/regression/...
```

Comment:
We now have a dedicated directory for regression tests, so it's easy to add more cases in the future.  
It also makes it very clear what this test does and why it's there.

Reply:
This does not appear to be invoked in any CI, specifically, this is only included in `test`, which is something we only do locally.

Reply:
I did not realized that we did not call make test in the CI :-( 
Since we have a specific action for unit tests and can, in the future, expand the regression tests to cover other scenarios, I added a new action to call this one.